clang/test/SemaSYCL/intel-max-work-group-size-ast.cpp

```diff
@@ -96,20 +96,6 @@ int main() {
     h.single_task<class test_kernel2>(
         []() [[intel::max_work_group_size(8, 8, 8)]] {});
 
-    // CHECK-LABEL: FunctionDecl {{.*}}test_kernel3
-    // CHECK:       SYCLIntelMaxWorkGroupSizeAttr
-    // CHECK-NEXT:  ConstantExpr{{.*}}'int'
-    // CHECK-NEXT:  value: Int 2
-    // CHECK-NEXT:  IntegerLiteral{{.*}}2{{$}}
-    // CHECK-NEXT:  ConstantExpr{{.*}}'int'
-    // CHECK-NEXT:  value: Int 2
-    // CHECK-NEXT:  IntegerLiteral{{.*}}2{{$}}
-    // CHECK-NEXT:  ConstantExpr{{.*}}'int'
-    // CHECK-NEXT:  value: Int 2
-    // CHECK-NEXT:  IntegerLiteral{{.*}}2{{$}}
-    h.single_task<class test_kernel3>(
```

Comment:
I would assume because we don't need attribute propagation? Can you remove `func_do_not_ignore `definition as well?

Reply:
>>I would assume because we don't need attribute propagation?

We only do attribute propagation in sycl 2017. 

>>Can you remove func_do_not_ignore definition as well?

Agreed with @elizabethandrews. Please remove this.